awscli/customizations/s3/results.py

```diff
@@ -400,23 +404,33 @@ def _add_progress_if_needed(self):
 
     def _print_progress(self, **kwargs):
         # Get all of the statistics in the correct form.
-        bytes_completed = human_readable_size(
-            self._result_recorder.bytes_transferred +
-            self._result_recorder.bytes_failed_to_transfer
-        )
-        expected_bytes_completed = human_readable_size(
-            self._result_recorder.expected_bytes_transferred)
         remaining_files = str(
             self._result_recorder.expected_files_transferred -
             self._result_recorder.files_transferred
         )
 
         # Create the display statement.
-        progress_statement = self.PROGRESS_FORMAT.format(
-            bytes_completed=bytes_completed,
-            expected_bytes_completed=expected_bytes_completed,
-            remaining_files=remaining_files
-        )
+        if self._result_recorder.expected_bytes_transferred > 0:
```

Comment:
Might be good to make a comment on the reasoning why we make this if statement (i.e. explain that some transfers have zero size and it would be strange to display 0/0 Bytes)